tools/accuracy_checker/accuracy_checker/adapters/classification.py

```diff
@@ -67,3 +70,15 @@ def process(self, raw, identifiers=None, frame_meta=None):
             result.append(single_prediction)
 
         return result
+
+    def _extract_predictions(self, outputs_list, is_multi_infer=False):
```

Comment:
please use common method signature. In some cases, it can be necessary to get some extra info from meta for correct multi infer processing, it is the reason why meta provided to this method.